randovania/games/am2r/json_data/Industrial Complex.txt

```diff
@@ -1545,22 +1545,27 @@ Extra - minimap_data: [{'x': 57, 'y': 30}, {'x': 57, 'y': 31}, {'x': 58, 'y': 30
       Any of the following:
           Space Jump or Can Use Spider Ball
           # Septogg
-          Power Beam and After Metroids Area 3 - Breeding Grounds Middle Bottom Gamma and Enabled Septogg Helpers
+          After Metroids Area 3 - Breeding Grounds Middle Bottom Gamma and Enabled Septogg Helpers
           # IBJ
           After Metroids Area 3 - Breeding Grounds Middle Bottom Gamma and Infinite Bomb Jumping (Advanced) and Can Use Bombs
+          # CBJ
```

Comment:
General comment: could the `after metroid` event be grouped at all? Also noticing it repeats in each method minus the CBJ, is that intentional?

Reply:
Adjusted

Reply:
Does the IBJ and CBJ not need to be `after metroid or movement`?

Reply:
The single IBJ (i.e. tap....tap....tap....tap....) is in there
The double IBJ (i.e tap-tap..tap-tap..tap-tap..) is not.


CBJ doesnt need to be in there